types/part_set.go

```diff
@@ -31,6 +32,11 @@ func (part *Part) ValidateBasic() error {
 	if len(part.Bytes) > int(BlockPartSizeBytes) {
 		return fmt.Errorf("too big: %d bytes, max: %d", len(part.Bytes), BlockPartSizeBytes)
```

Comment:
I know this is not strictly necessary, but we can turn this `fmt.Error` into a proper error as we're doing below?